Comanche055/IMU_MODE_SWITCHING_ROUTINES.agc

```diff
@@ -401,12 +402,12 @@ PIPFREE2	TC	IBNKCALL	# ISS WARNING MIGHT COME ON NOW.
 
 		TCF	MODEEXIT
 
-PIPFREE		INHINT			# PROGRAM DONE WITH PIPAS.  DON'T LIGHT
+PIPFREE		INHINT			# PROGRAM DONE WITH PIPAS.  DONT LIGHT
 		CS	IMODES30	# ISS WARNING.
 		MASK	BIT1
 		ADS	IMODES30
 
-		MASK	BIT10		# IF PIP FAIL ON, DO PROG ALARM AND RESET
+		MASK	BIT10		# IF PIP FAIL ON, DO PROG ALSRM AND RESET
```

Comment:
Dude. What is with all these mispellings? I see your a Polish national (
Witaj! Mój ojciec jest z Wrocławia); if English isn't your primary language, don't make English corrections.